Documentation/cheatsheet.rst

```diff
@@ -2,157 +2,396 @@
+documentation <https://kubernetes.io/docs/reference/kubectl/jsonpath/>`_
 
 ::
 
-        # Get the current agent configuration
+    $ cilium endpoint list -o jsonpath='{[*].id}'
+    29898 38939 56326
+    $ cilium endpoint list -o jsonpath='{range [*]}{@.id}{"="}{@.status.policy.spec.policy-enabled}{"\n"}{end}'
+    29898=none
+    38939=none
+    56326=none
 
-        cilium config
 
-Policy management
+Shell Tab-completion
+--------------------
+
+If you use bash or zsh, Cilium CLI can provide tab completion for subcommands.
+If you want to install tab completion, you should run the following command in
+your terminal.
+
+::
+
+   $ source <(cilium completion)
+
+If you want to have Cilium completion always loaded, you can install using the
+following:
+
+::
+
+    $ echo "source <(cilium completion)" >> ~/.bashrc
+
+
+Command examples:
 =================
 
+Basics
+------
+
+Check the status of the agent
 ::
 
-	# Importing a policy
+    $ cilium status
+    KVStore:                Ok         Consul: 172.17.0.3:8300
+    ContainerRuntime:       Ok
+    Kubernetes:             Disabled
+    Cilium:                 Ok         OK
+    NodeMonitor:            Listening for events on 2 CPUs with 64x4096 of shared memory
+    Cilium health daemon:   Ok
+    Controller Status:      6/6 healthy
+    Proxy Status:           OK, ip 10.15.28.238, port-range 10000-20000
+    Cluster health:   1/1 reachable   (2018-04-11T07:33:09Z)
+    $
+
+Get a detailed status of the agent:
+::
+
+    $ cilium status --all-controllers --all-health --all-redirects
+    KVStore:                Ok         Consul: 172.17.0.3:8300
+    ContainerRuntime:       Ok
+    Kubernetes:             Disabled
+    Cilium:                 Ok         OK
+    NodeMonitor:            Listening for events on 2 CPUs with 64x4096 of shared memory
+    Cilium health daemon:   Ok
+    Controller Status:      6/6 healthy
+      Name                                 Last success   Last error   Count   Message
+      kvstore-lease-keepalive              2m52s ago      never        0       no error
+      ipcache-bpf-garbage-collection       2m50s ago      never        0       no error
+      resolve-identity-29898               2m50s ago      never        0       no error
+      sync-identity-to-k8s-pod (29898)     50s ago        never        0       no error
+      sync-IPv4-identity-mapping (29898)   2m49s ago      never        0       no error
+      sync-IPv6-identity-mapping (29898)   2m49s ago      never        0       no error
+    Proxy Status:   OK, ip 10.15.28.238, port-range 10000-20000
+    Cluster health:         1/1 reachable   (2018-04-11T07:32:09Z)
+      Name                  IP              Reachable   Endpoints reachable
+      runtime (localhost)   10.0.2.15       true        false
+    $
+
+Get the current agent configuration
+::
 
-	cilium policy import <my-policy.json>
+    cilium config
+
+Policy management
+-----------------
 
+
+Importing a policy
```

Comment:
Since this is a cheat sheet, it might be good to mention what kind of policy this imports.
"Import a Cilium Network Policy"
or a note saying it isn't a k8s NP